packages/perseus/src/types.js

```diff
@@ -351,13 +350,11 @@ export type PerseusDependencies = {|
 
     // KaTeX related dependencies
     getKaTeX: () => Promise<katex>,
-    getRenderA11yString: () => Promise<katexA11y>,
-    loadMathjax: () => Promise<mixed>,
+    // loadMathjax is deprecated. Perseus doesn't call it.
+    loadMathjax?: () => Promise<mixed>,
     logKaTeXError: (expression: string, error: Error) => Promise<?Response>,
-    KatexProvider: React.ComponentType<{|children: React.Node|}>,
```

Comment:
In theory, removing `KatexProvider` is a breaking change, since clients who are passing it will get type errors. Probably better to make it optional.